Source/WebKit/Shared/CoordinatedGraphics/threadedcompositor/ThreadedCompositor.cpp

```diff
@@ -375,6 +375,8 @@ WebCore::DisplayRefreshMonitor& ThreadedCompositor::displayRefreshMonitor() cons
 
 void ThreadedCompositor::frameComplete()
 {
+    WTFEmitSignpost(this, frameComplete);
```

Comment:
Small nit, but all marks start with an uppercase character, and I guess it's better to keep the pattern here? e.g. `FrameComplete`

Reply:
Agreed, will fix!